index.html

```diff
@@ -72,6 +73,15 @@
 				Inner Right Sidebar
 			</section>
 
+			<ts-note
+				id="rootNote"
+				icon="info2"
+				buttons='[{"label": "Button", "id": "noteId"}, {"label": "Button 2", "type": "secondary"}]'
```

Comment:
How many buttons could we support? 
How could we handle `onClick` It is hard for developer transfer an function into json.

Reply:
Since there is none in the `ts-ui` I didn't put any limitation here as well! Should I add a limitation?
They don't need to put function in the json, they can listen to `button-click` event, and in one function handle all the button clicks! like this :
https://github.com/Tradeshift/elements/pull/125/files/77015fbf42f60c349ed664381819441c6cc454db#diff-eacf331f0ffc35d4b482f1d15a887d3bR183

Reply:
OK, Cool. Let's keep it like this right now. I think one button is enough.